### DISCUSSION TITLE
Problem with publish a react (next) project.

### DISCUSSION THREAD

**Discussion by arteclaw-io** [community/community#77004] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I'm new to GH Pages. This is my site, I just created uploading all my files from another project to a new repo. I'm not being able to see my app working, just a static page.
https://arteclaw-io.github.io/
I got this issue when deploying from vs code:
info  - Creating an optimized production build ..
> deploy
> gh-pages -d build

ProcessError: remote: Permission to arteclaw-io/arteclaw-io.github.io.git denied to fran-cisc0.
fatal: unable to access 'https://github.com/arteclaw-io/arteclaw-io.github.io.git/': The requested URL returned error: 403

    at ChildProcess.<anonymous> (/Users/danielarias/GitHub/arteclaw-io.github.io/node_modules/gh-pages/lib/git.js:42:16)
    at ChildProcess.emit (node:events:514:28)
    at maybeClose (node:internal/child_process:1105:16)
    at ChildProcess._handle.onexit (node:internal/child_process:305:5)
    
I pushed all my files, thinking that it will be automatically deployed but maybe I'm missing something.
Thanks in advance.

**Comment by kur0neko** [comment]
I'm unsure if this is a problem with the GitHub token,  you should edit the script file 

   `run: |
        node deployment-script.js
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}`

Edit 

`https://${process.env.GITHUB_TOKEN}@github.com/<repo of this project>/<name git>.git`
↓
`https://x-access-token:${process.env.GITHUB_TOKEN}@github.com/<repo of this project>/<name git>.git`
